index.html

```diff
@@ -179,22 +182,21 @@
       <p>
         With an active, public development roadmap, seL4 continues to solidify its
         position as the leading secure operating system and the industry
-        standard for verified software.
+        standard for verified software. Evolution is a constant in the seL4 ecosystem.
       </p>
       <p>
-        As seL4 gains wider adoption, it continues to evolve, expanding its
-        support for an increasing range of platforms, architectures,
-        configurations and features.
+        seL4 itself is expanding its support for an increasing range
+        of platforms, architectures, configurations and features.
       </p>
       <p>
-        The ecosystem is also expanding, with the development of frameworks,
+        The ecosystem is expanding with the development of frameworks,
         tools, components and language support to facilitate the production of
         seL4-based systems.
       </p>
       <p>
-        The verification, which makes seL4 unique, evolves alongside seL4. The
-        formal proofs are constantly maintained and kept in lock-step with the
-        seL4 code.
+        Formal verification makes seL4 unique and is evolving alongside seL4.
+        The formal proofs are constantly maintained, improved, and kept in
+        lock-step with the seL4 code.
```

Comment:
Suggested simplification:

The formal proofs, which makes seL4 unique, are evolving alongside seL4. They are constantly maintained, improved, and kept in lock-step with the seL4 code.

Reply:
I'd suggest that "The formal proofs, which make seL4 unique, evolve alongside seL4" is punchier, but I'm really nitpicking now :-)